sonar-helm-for-iac/build.gradle.kts

```diff
@@ -49,8 +49,7 @@ if (isCi) {
             "go.mod",
             "go.sum",
             "make.bat",
-            "make.sh",
-            "template-evaluation.proto")
```

Comment:
Was it (`protoc`) done twice before this PR? Once in  `sonar-helm-for-iac` and once in the kubernetes extension?

Reply:
No, there is still no dependency on `compileProtobufGo` task, because we commit the generated code, so need to run it manually anyway. I tried to add this dependency to have build be more consistent, but for some reason caching of task results was not working, so I left it as is for now.